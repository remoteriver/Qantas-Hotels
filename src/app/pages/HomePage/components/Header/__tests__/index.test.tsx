import { render } from '@testing-library/react';
import { createRenderer } from 'react-test-renderer/shallow';
import { Header } from '..';

const renderer = createRenderer();

describe('<HotelCard />', () => {
  beforeAll(() => {});

  it('should render and match the snapshot', () => {
    //act
    const component = render(<Header></Header>);

    //assert
    expect(component.baseElement).toMatchSnapshot();
  });
});
