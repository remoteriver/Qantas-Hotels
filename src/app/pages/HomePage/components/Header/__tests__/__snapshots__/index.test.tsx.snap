// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HotelCard /> should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiToolbar-root MuiToolbar-regular"
    >
      <img
        alt="logo"
        class="makeStyles-logo-1"
        src="img/qantas-logo.png"
      />
    </div>
  </div>
</body>
`;
